integration/src/binance/tx02.mainnet.signed.json

```diff
@@ -1,40 +1,36 @@
 {
-  "chain_id": "Binance-Chain-Tigris",
+  "account_number": "471113",
+  "chain_id": "Binance-Chain-Nile",
   "data": null,
   "memo": "",
   "msgs": [
     {
       "inputs": [
         {
-          "address": {
-            "type": "Buffer",
-            "data": [234, 93, 122, 233, 154, 156, 226, 249, 219, 124, 67, 169, 7, 55, 112, 36, 183, 243, 67, 232]
-          },
+          "address": "bnb1afwh46v6nn30nkmugw5swdmsyjmlxslgjfugre",
           "coins": [
             {
-              "denom": "BNB",
-              "amount": "1000"
+              "amount": "0.00001",
+              "denom": "BNB"
             }
           ]
         }
       ],
       "outputs": [
         {
-          "address": {
-            "type": "Buffer",
-            "data": [103, 156, 216, 31, 171, 65, 66, 104, 64, 250, 96, 40, 9, 112, 81, 59, 129, 14, 180, 30]
-          },
+          "address": "bnb1v7wds8atg9pxss86vq5qjuz38wqsadq7e5m2rr",
           "coins": [
             {
-              "denom": "BNB",
-              "amount": "1000"
+              "amount": "0.00001",
+              "denom": "BNB"
             }
           ]
         }
-      ],
-      "msgType": "MsgSend"
+      ]
     }
   ],
+  "sequence": "17",
+  "source": "0",
```

Comment:
These values of sequence and data type match the ones actually signed -- which do not, until 2cb9480, actually come from here, but from an call out to the interwebs to check actual on-chain state of this address.